tests/test_formparsers.py

```diff
@@ -36,16 +38,17 @@ async def app(scope, receive, send):
 async def multi_items_app(scope, receive, send):
     request = Request(scope, receive)
     data = await request.form()
-    output = {}
+    output: typing.Dict[str, list] = {}
     for key, value in data.multi_items():
         if key not in output:
             output[key] = []
         if isinstance(value, UploadFile):
             content = await value.read()
+            content = content.decode() if isinstance(content, bytes) else content
```

Comment:
Here `content` is a `Union[str,bytes]` and str won't have the decode method.

Reply:
Maybe #1312 is relevant?

Reply:
Hmm..I think they might be. Do you think this can be a blocker?

Reply:
I think it would be nice to merge that first since you're making changes here to work around the problem. FYI I marked #1312 as breaking changes only because the use of str is in the public docs. I'm fairly certain there are zero people using it that way though, so really I don't think we should treat it as a break and change

That said this is just typing for tests, I don't think it's worth holding up if it's a lot of trouble.

Reply:
Now that #1312 is merged, can you also review this? @adriangb @tomchristie